php/router.php

```diff
@@ -0,0 +1,105 @@
+<?php
+// Used by `wp server` to route requests.
```

Comment:
> They are not top-level, but inside the WP_CLI\Router namespace.

Sorry. To clarify, when I said "top-level", I meant "functions other WP-CLI developers have easier access to." Because, in some senses, we're making them a public API.

> If you have specific questions that can't be answered by the current comments, I'd be glad to answer them.

Sure. For each, why?